test/parallel/test-net-socket-write-error.js

```diff
@@ -8,13 +8,14 @@ const server = net.createServer().listen(0, connectToServer);
 
 function connectToServer() {
   const client = net.createConnection(this.address().port, () => {
-    assert.throws(() => client.write(1337),
-                  {
-                    code: 'ERR_INVALID_ARG_TYPE',
-                    name: 'TypeError'
-                  });
-
-    client.destroy();
+    client.write(1337, common.expectsError({
```

Comment:
Seems like `common` is not defined: https://ci.nodejs.org/job/node-test-commit-linuxone/18077/nodes=rhel7-s390x/testReport/junit/(root)/test/parallel_test_net_socket_write_error/

I suggest to run the linter before pushing :) I very well know that it takes a while to always run the tests but the linter already catches issues like these.